### PR TITLE
build(java): generate docfx yml on release

### PR DESCRIPTION
**Pull request by eaball35**

### CODE REVIEW

File: synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc11.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Copyright 2019 Google Inc.
```
Comment (lesv) [inline synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc11.sh:2]: This is new, should be 2021 Google LLC

**Review by lesv — APPROVED** [review]
Please fix the copyright in your script, then LGTM

File: synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc11.cfg [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# Configure the docker image for kokoro-trampoline
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java11"
+}
+
+env_vars: {
+  key: "TRAMPOLINE_BUILD_FILE"
+  value: "github/java-memcache/.kokoro/release/publish_javadoc11.sh"
```
Comment (chingor13) [inline synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc11.cfg:17]: If the 2 scripts are the same and the only differences are the docker image and the bucket name, we can re-use the same shell script.

Comment (chingor13) [reply synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc11.cfg:17]: Actually it looks like the `destination-prefix` option is new. You could potentially make that configurable via an env variable or keep the 2 separate scripts.

Comment (eaball35) [reply synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc11.cfg:17]: Makes sense, I think I'll just leave as is unless you have a strong opinion about this

**Pull request merged**